cron/run_benchmarks.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/bash
+echo "Running benchmark update `date`"
+BENCHMARK_REPO=${BENCHMARK_REPOSITORY:-$HOME/dask-benchmarks}
+DASK_DIR=$BENCHMARK_REPO/dask
+DISTRIBUTED_DIR=$BENCHMARK_REPO/distributed
+DASK_CONFIG=${DASK_ASV_CONFIG:-$HOME/asv.dask.conf.json}
+DISTRIBUTED_CONFIG=${DISTRIBUTED_ASV_CONFIG:-$HOME/asv.distributed.conf.json}
+
+source activate dask-asv
+
+echo "Updating benchmark repo..."
+cd $BENCHMARK_REPO
+git checkout master
+git pull
+
+echo "Running dask benchmarks..."
+cd $DASK_DIR
+asv --config $DASK_CONFIG run NEW
+DASK_STATUS=$?
+if [ "$DASK_STATUS" -eq "0" ]; then
+  echo "Generating dask html files..."
+  asv --config $DASK_CONFIG publish
+fi
+
+echo "Running distributed benchmarks..."
+cd $DISTRIBUTED_DIR
+asv --config $DISTRIBUTED_CONFIG run NEW
+DISTRIBUTED_STATUS=$?
+if [ "$DISTRIBUTED_STATUS" -eq "0" ]; then
+  echo "Generating distributed html files..."
+  # Currently install dask dependency for distributed via pip install git+http to
+  # get current dask master. asv does not directly support this even though you
+  # can get it to work. However directory structure gets messed up and machine.json
+  # is not in the correct location to generate the graphs. Thus this hack to copy it
+  # to the right locations before running publish.
+  find /home/ec2-user/results/distributed/aws-ec2-c4.xlarge -type d -exec cp /home/ec2-user/results/distributed/aws-ec2-c4.xlarge/machine.json {} \;
```

Comment:
This whole line is just a workaround to install dask master. I have an open issue for asv here: https://github.com/spacetelescope/asv/issues/494.  Plan to take a look at applying pv's fix.